types/index.d.ts

```diff
@@ -373,6 +379,23 @@ export namespace OperationOptions {
     account: string;
     signer: SignerKeyOptions;
   }
+  interface Clawback extends BaseOptions {
+    asset: Asset;
+    amount: string;
+    from: string;
+  }
+  interface ClawbackClaimableBalance extends BaseOptions {
+    balanceId: string;
+  }
+  interface SetTrustLineFlags extends BaseOptions {
+    trustor: string;
+    asset: Asset;
+    flags: {
+      authorized?: boolean | undefined;
```

Comment:
If we mark them as optional, should we make corresponding changes to the [documentation](https://github.com/stellar/js-stellar-base/blob/c687cf9411/src/operations/set_trustline_flags.js#L21-L27)?

```js
 * @param {bool}   [opts.flags.authorized]  - authorize account to perform transactions
 *     with its credit
 * @param {bool}   [opts.flags.authorizedToMaintainLiabilities] - authorize
 *     account to maintain and reduce liabilities for its credit
 * @param {bool}   [opts.flags.clawbackEnabled] - stop claimable balances on this
 *     trustlines from having clawbacks enabled (this flag can only be set to
 *     false!)
```

Reply:
Good call, done in 1f91b83fcf114e339f7c894e9397a5073ed6306d.